Source/swiftlint/StructuredText.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  StructuredText.swift
+//  SwiftLint
+//
+//  Created by Chris Eidhof on 20/05/15.
+//  Copyright (c) 2015 Realm. All rights reserved.
+//
+
+import Foundation
+
+typealias StructuredInlineText = String
+
+private extension Array {
+    func dropWhile(x: Element -> Bool) -> [Element] {
+        var count = 0
+        while x(self[count]) { count++ }
+        var copy = self
+        copy.removeRange(0..<count)
+        return copy
+    }
+}
+
+
+extension String {
+    var chomped: String {
+        return String(reverse(self).dropWhile { $0 == "\n" }.reverse())
```

Comment:
Oh, actually I'm a dummy. But you should use `stringByTrimmingTrailingCharactersInSet()` defined in `String+SourceKitten`.